source/activity/view-logs.txt

```diff
@@ -106,25 +200,25 @@ endpoint with either or both of the ``start_date`` and ``end_date``
 fields:
 
 .. note:: Result Pagination
-   
+
    If the date range that you specify includes more than 100 log
    entries, you will need to :ref:`run paginated queries
    <logging-get-paginated-logs>` to access all of the entries.
```

Comment:
might just be me, but I get slightly annoyed by inline links taking you somewhere on the same page when I'm not expecting it (and can't hit back to return to where you were...). maybe revise to make it more clear you're linking to the `Get Paginated Logs` section on this same page?